metalConfig.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: metallb.io/v1beta1
```

Comment:
Adding a sample for metallb is a good thing, but it should go to a documentation folder because meatllb is not the default standard for pihole. The user can choose which lb to use.